src/common/crypto/tinycrypt/inc/ecc.h

```diff
@@ -0,0 +1,462 @@
+/* ecc.h - TinyCrypt interface to common ECC functions */

```

Comment:
Can we move these source files to `third_party`?

Reply:
Done, thanks